packages/zoe/src/contracts/coveredCall.js

```diff
@@ -7,64 +7,78 @@ import { makeZoeHelpers } from '../contractSupport';
 
 const rejectMsg = `The covered call option is expired.`;
 
-/** @typedef {import('../zoe').ContractFacet} ContractFacet */
+/**
+ * In a covered call, the owner of a digital asset sells a call
+ * option. A call option is the right to buy the digital asset at a
+ * pre-determined price, called the strike price. The call option has an expiry
+ * date, at which point the contract is cancelled.
+ *
+ * In this contract, the expiry date is represented by the deadline at
```

Comment:
Do you really need the "represented by"? Or is the expiry date just the deadline and you can drop the "r b"

Reply:
done